schemas/com.snowplowanalytics.snowplow/media_player_event_ad_break_start/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event that signals the start of an ad break.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_event_ad_break_start",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "label": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "A custom identifier for the media player",
```

Comment:
If this is reused across ad break start and end shouldn't the customer identifier just be added as a context?

Reply:
I think that makes sense – the `label` relates to the media player, not directly the event so I think it should be in that entity. I'll make a new version of the entity and move the property there.

Reply:
✅ removed `label` from all events here and moved to a new `player` entity (previously `media_player`) in the other entities PR.